integration-libs/opf/base/root/facade/opf-order.facade.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { facadeFactory } from '@spartacus/core';
+import { Order } from '@spartacus/order/root';
+import { Observable } from 'rxjs';
+import { OPF_BASE_FEATURE } from '../feature-name';
+
+@Injectable({
+  providedIn: 'root',
+  useFactory: () =>
+    facadeFactory({
+      facade: OpfOrderFacade,
+      feature: OPF_BASE_FEATURE,
+      methods: ['placeOpfOrder'],
+    }),
+})
+export abstract class OpfOrderFacade {
+  /**
+   * Places an order
+   */
+  abstract placeOpfOrder(termsChecked: boolean): Observable<Order>;
```

Comment:
[MEDIUM]

For easier reasoning about this method when seeing its usage, let's change the boolean parameter to an object with a semantic key: `{ termsChecked: boolean }`.

See difference when a random dev sees the usage for the first time:
```ts
OpfOrderFacade.placeOpfOrder(true); // Developer thinks: what does `true` mean? I have to lookup the implementation of the method or JSDocs.
```
vs
```
OpfOrderFacade.placeOpfOrder({termsChecked: true});
```